roles/openshift_facts/library/openshift_facts.py

```diff
@@ -395,6 +395,23 @@ def set_metrics_facts_if_unset(facts):
             facts['common']['use_cluster_metrics'] = use_cluster_metrics
     return facts
 
+def set_openshift_metrics_facts_if_unset(facts):
+    """ Set openshift metrics facts if not already present in facts dict
+            dict: the facts dict updated with the generated openshift metrics facts if
+            missing
+        Args:
+            facts (dict): existing facts
+        Returns:
+            dict: the facts dict updated with the generated openshift metrics
+            facts if they were not already present
+
+    """
+    if 'common' in facts:
+        if 'use_cluster_metrics_hawkular' not in facts['common']:
+            use_cluster_metrics_hawkular = False
+            facts['common']['use_cluster_metrics_hawkular'] = use_cluster_metrics_hawkular
```

Comment:
A few suggestions here:
- Since this is a static "fact", it should be set in the get_defaults method. set_*_if_unset methods were originally introduced to solve setting facts based on the runtime value of other facts and has since been used for simple facts (which we are working on cleaning up).
- From the previous use, it looks like use_cluster_metrics_hawkular should be use_openshift_metrics. I'm not a huge fan of that naming though...  possibly openshift_hosted_metrics_deploy=True/False for the user exposed variable, and the fact could be openshift.hosted.metrics.deploy ? @abutcher